src/_plugins/active_nav_generator.rb

```diff
@@ -0,0 +1,121 @@
+# Generates and stores active entry data for paths
+# based on the data declared in `_data/sidenav.yml`.
+# This stored active entry data is used in `active_nav_for_page`
+# and the `active_nav_for_page` filter it declares to
+# determine which entries to indicate as active in the sidenav
+# on each page.
+module ActiveNavGenerator
```

Comment:
This doesn't seem like it should be in the diff, since https://github.com/flutter/website/pull/9061 has been merged. Do you know why this is showing up?